client_core/src/com/mygdx/game/client_core/model/GameState.java

```diff
@@ -21,14 +24,18 @@ public class GameState {
   private final ComponentMapper<Coordinates> coordinatesMapper;
   private final ComponentMapper<SubField> subFieldMapper;
```

Comment:
W ogóle to ja chcę tego gamestate usunąć z klienta, albo ograniczyć go jakoś, więc wolałbym, żebyś nie pisał w nim.

Reply:
i coś dopisałeś :V